host/xtest/xtest_4000.c

```diff
@@ -4260,7 +4260,7 @@ static void xtest_tee_test_4009(ADBG_Case_t *c)
 	uint8_t out[2048];
 	size_t out_size;
 	uint32_t size_bytes;
-	int i;
+	uint32_t i;
```

Comment:
Any special reason for a 32-bit type? A `unsigned` or `size_t` would be a more natural choice.